wurst/systems/entities/bosses/DiscoDuck.wurst

```diff
@@ -56,6 +58,10 @@ public class DiscoDuck extends Hostile
         let pos = getPos()
         let pinion = pinions.placeRandomItem(pos)
         pinions.removeItemType(pinion.getTypeId())
+        // TODO : Wrap this up in a Boss entity class?
+        PlaySound(Sounds.warning)
+        printTimed(("The " + getUnit().getName() + " has been killed!").color("red".toColor()), 5)
```

Comment:
This should use `format`.

I don't think it needs to go into `Boss` if you don't want to alert for every boss, but you shouldn't repeat the same logic in two different files. You could just keep it simple and add an event for unit death directly and check it against a custom set of IDs.